instrumentation/metric/histogram.go

```diff
@@ -69,3 +72,21 @@ func (h *Histogram) Export() interface{} {
 		h.histo.TotalCount(),
 	}
 }
+
+func (h *Histogram) Reset() {
+	h.histo.Reset()
```

Comment:
check whether this makes sense - the library has a `WindowedHistogram` for this purpose